app/javascript/components/shared/stylings/DeleteModal.jsx

```diff
@@ -0,0 +1,36 @@
+import React, { useState } from 'react';
+import { Modal, Button } from 'react-bootstrap';
+import PropTypes from 'prop-types';
+
+export default function DeleteModal({ title, body, children }) {
+  const [show, setShow] = useState(false);
+
+  const handleClose = () => setShow(false);
+  const handleShow = () => setShow(true);
+
+  return (
+    <>
+      <Button variant="danger" onClick={handleShow}>
+        Delete
+      </Button>
+
+      <Modal className="text-center" show={show} onHide={handleClose}>
```

Comment:
I think it would be better if you create a generic `Modal` (Header, Title, Body,Footer, states, ...) component that can be used afterwards in the application and use it in the `DeleteModal` adding the elements necessary for the deletion context like the **Delete** button etc...
Also it may be better if the children prop is rendered in the body section and and not in the footer and make the footer itself a prop `<Modal title={title} footer={footer}> {body} </Modal>`.

Reply:
I agree for a generic Modal - I will work on this right now.
I don't understand what you mean by _making the footer a prop._
Maybe we should wait for mock ups before really digging into it.

Reply:
I renamed the DeleteModal component to Modal.
The logic for putting the children (the Form) in the Modal Footer was that I (assumed incorrectly) that most of the Modal Forms would be a single button. 
But after some thinking, we should wait for more mock ups before spending time on customising the Modal.